cordova-lib/src/cordova/plugin.js

```diff
@@ -305,6 +289,49 @@ module.exports = function plugin(command, targets, opts) {
     });
 };
 
+function determinePluginTarget(projectRoot, cfg, target, fetchOptions) {
+    var parts = target.split('@');
+    var id = parts[0];
+    var version = parts[1];
+
+    // If no version is specified, retrieve the version (or source) from config.xml
+    if (version || cordova_util.isUrl(id) || cordova_util.isDirectory(id)) {
+        return Q(target);
+    } else {
```

Comment:
Minor nit: superfluous `else` - remove it, and decrease the indentation level of the rest of the function.